internal/engine/upper_test.go

```diff
@@ -836,15 +891,17 @@ func makeFakePodWatcherMaker(ch chan *v1.Pod) func(context.Context, *store.Store
 
 type testFixture struct {
 	*tempdir.TempDirFixture
-	ctx        context.Context
-	cancel     func()
-	upper      Upper
-	b          *fakeBuildAndDeployer
-	fsWatcher  *fakeNotify
-	timerMaker *fakeTimerMaker
-	docker     *docker.FakeDockerClient
-	hud        *hud.FakeHud
-	podEvents  chan *v1.Pod
+	ctx                   context.Context
+	cancel                func()
+	upper                 Upper
+	b                     *fakeBuildAndDeployer
+	fsWatcher             *fakeNotify
+	timerMaker            *fakeTimerMaker
+	docker                *docker.FakeDockerClient
+	hud                   *hud.FakeHud
+	podEvents             chan *v1.Pod
+	createManifestsResult chan error
+	log                   *bytes.Buffer
```

Comment:
fwiw, i've been using a ThreadSafeBuffer to make this a bit easier to check the buffer without stopping